### PR TITLE
Add docstring for default-middlewares

### PR DESCRIPTION
**Pull request by MalloZup**
# what does this pR?
- add small documentation about highlevel usage of the default-middlwares var

### CODE REVIEW

File: src/clojure/nrepl/server.clj [hunk 1]
```diff
@@ -79,6 +79,12 @@
   (t/send transport (response-for msg :status #{:error :unknown-op :done} :op op)))
 
 (def default-middlewares
+  "middleware functions vars that are implicitly merged with any user-specified
+   middleware provided to nrepl.server/default-handler
+
+   For new middleware, use the convention `wrap-` prefix.
```
Comment (bbatsov) [inline src/clojure/nrepl/server.clj]: I think this is not the place to mention naming conventions.

File: src/clojure/nrepl/server.clj [hunk 2]
```diff
@@ -79,6 +79,12 @@
   (t/send transport (response-for msg :status #{:error :unknown-op :done} :op op)))
 
 (def default-middlewares
+  "middleware functions vars that are implicitly merged with any user-specified
```
Comment (bbatsov) [inline src/clojure/nrepl/server.clj]: Please change this to:

`Middleware vars that are implicitly merged with any user-specified
   middlewares provided to `nrepl.server/default-handler`.`

File: src/clojure/nrepl/server.clj (hunk 1 above)
Comment (MalloZup) [reply src/clojure/nrepl/server.clj]: ok!

File: src/clojure/nrepl/server.clj (hunk 2 above)
Comment (MalloZup) [reply src/clojure/nrepl/server.clj]: ok

**Comment by MalloZup** [comment]
@bbatsov feel free to take look

doh

**Comment by bbatsov** [comment]
Thanks!

**Pull request merged**